cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceAction.scss

```diff
@@ -125,6 +125,7 @@ $cell-height: 2.5em;
                   top: 0.5em;
                   left: 2em;
                   cursor: pointer;
+                  color: #0275d8;
```

Comment:
we should probably externalize this. We should minimize the amount of colors we use and we should use variables for reuse everywhere.